tests/app/conftest.py

```diff
@@ -2462,6 +2467,23 @@ def mock_va_profile_response():
         return json.load(f)
 
 
+@pytest.fixture
+def x_minutes_ago():
```

Comment:
You created a function `5_minutes_ago` somewhere above.  Should you use this instead?

Reply:
Got rid of five_minutes_ago. I meant to replace them all with x_minutes_ago because it's more versatile. Cut out the five_minutes_ago method.